frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -110,8 +184,79 @@ const filters = [
   },
 ];
 
-const NodesPage = (props) => (
-  <ListPage {...props} ListComponent={NodesTable} rowFilters={filters} />
-);
+const fetchNodeMetrics = (): Promise<NodeMetrics> => {
+  const metrics = [
+    {
+      key: 'usedMemory',
+      query: 'sum by (instance) (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes)',
+    },
+    {
+      key: 'totalMemory',
+      query: 'sum by (instance) (node_memory_MemTotal_bytes)',
+    },
+    {
+      key: 'usedStorage',
+      query: 'sum by (instance) (node_filesystem_size_bytes - node_filesystem_free_bytes)',
+    },
+    {
+      key: 'totalStorage',
+      query: 'sum by (instance) (node_filesystem_size_bytes)',
+    },
+    {
+      key: 'cpu',
+      query: 'sum by(instance) (instance:node_cpu:rate:sum)',
+    },
+    {
+      key: 'pods',
+      query: 'sum by(node)(kubelet_running_pod_count)',
+    },
+  ];
+  const promises = metrics.map(({ key, query }) => {
+    const url = getPrometheusURL({ endpoint: PrometheusEndpoint.QUERY, query });
+    return coFetchJSON(url).then(({ data: { result } }) => {
+      return result.reduce((acc, data) => {
+        const value = Number(data.value[1]);
+        return _.set(acc, [key, data.metric.instance || data.metric.node], value);
+      }, {});
+    });
+  });
+  return Promise.all(promises).then((data: any[]) => _.assign({}, ...data));
+};
+
+const mapDispatchToProps = (dispatch): MapDispatchToProps => ({
+  setNodeMetrics: (metrics) => dispatch(setNodeMetrics(metrics)),
+});
+
+const showMetrics = PROMETHEUS_BASE_PATH && window.innerWidth > 1200;
+
+const NodesPage = connect<{}, MapDispatchToProps>(
+  null,
+  mapDispatchToProps,
+)((props: MapDispatchToProps) => {
+  const { setNodeMetrics: setMetrics } = props;
+
+  React.useEffect(() => {
+    const updateMetrics = () =>
+      fetchNodeMetrics()
+        .then(setMetrics)
+        .catch((e) => {
+          // esline-disable-next-line no-console
```

Comment:
I'm not sure how this passed the linter. `esline`?

Reply:
Maybe the linter rules are different for console-app. Can you remove this line with the typo if it's not needed?

Reply:
I will fix the typo. It would stop vscode from highlighting it with that ugly yellow line.

Reply:
> I will fix the typo. It would stop vscode from highlighting it with that ugly yellow line.

But the linter should have caught it. Either it will fail the linter, and we need to disable, or it won't. If VSCode is showing something different, it sounds like VSCode is not configured correctly.

We should remove this if it's not needed.

Reply:
eslint does not fail but it does warn us. 
![Screenshot from 2020-03-19 02-58-25](https://user-images.githubusercontent.com/54092533/77009210-8885d980-698d-11ea-9f39-b485522167df.png)

Reply:
Hmm. We should fix this.  Rules should be the same everywhere?

Reply:
@spadgett @benjaminapetersen 
https://github.com/openshift/console/pull/4773
This will increase the severity of no-console to error.

Reply:
@bipuladh Thanks, did you push a fix for the typo? I still see `esline`

Reply:
my bad. I forgot to push it. changed now.